.eslintrc

```diff
@@ -17,16 +23,22 @@
   "globals": {
     "YT": true,
   },
+  "settings": {
+    "import/core-modules": [
```

Comment:
what is that for?

Reply:
I removed the "import/no-extraneous-dependencies" rule, because it's important to check for it. This tells `eslint` that those modules are whitelisted to be imported under `devDependencies` even though they're not under `dependencies`. This is preferred since we could import a module that it's not listed in `package.json` at all and `eslint` wouldn't complain because of the "no-extraneous-dependencies"